_includes/author-profile.html

```diff
@@ -1,6 +1,6 @@
 {% include base_path %}
 
-{% if page.author %}
+{% if page.author and site.data.authors[page.author] %}
```

Comment:
That is exactly my thinking behind this line. I don't have an authors.yml file, so it was failing.  I added the check so that it would default back to the _config.yml settings.